src/bot/integrations/discord.ts

```diff
@@ -371,6 +385,27 @@ class Discord extends Integration {
     } catch (e) {
       warning(e.stack);
     }
+
+    try {
+      this.changeClientOnlinePresence();
+    } catch (e){
+      warning(e.stack);
+    }
+  }
+
+  @onChange('onlinePresence')
+  async changeClientOnlinePresence() {
+    try {
+      if (api.isStreamOnline) {
+        this.client?.user?.setStatus('online');
+        this.client?.user?.setPresence( { status: 'online', activity: {name: `${api.stats.currentTitle}`, type: 'STREAMING', url: `https://twitch.tv/${oauth.generalChannel}`} } );
+      } else {
+        this.client?.user?.setPresence( {status: this.onlinePresence, activity:{name:'',type:undefined,url:''}} );
```

Comment:
so we should have 

this.onlinePresence -> presence when stream is online
this.offlinePresence -> presence when stream is offline